src/EPR.Calculator.Frontend/wwwroot/js/Download.js

```diff
@@ -0,0 +1,35 @@
+﻿function downloadFile(url, errorAction, event) {
+    event.preventDefault();
+    $.ajax({
```

Comment:
You might have to introduce a Timeout parameter in the configuration and pass it as a parameter
This is needed as Downloading a file might take a while [Large Files]
I can help you with that in a call. Thanks